src/python/pants/backend/python/typecheck/mypy/rules.py

```diff
@@ -69,7 +69,7 @@ def generate_argv(
     file_list_path: str,
     python_version: Optional[str],
 ) -> Tuple[str, ...]:
-    args = [f"--python-executable={venv_python}", *mypy.args]
+    args = [f"--python-executable={venv_python}", "--no-error-summary", *mypy.args]
```

Comment:
This just silences the "Success: ..." or "Found 1 error in 1 file (checked 2 source files)" messages, not the specific error information in the event of errors.